internal/controller/thanosquery_controller.go

```diff
@@ -45,6 +45,10 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	queryComponent = "query"
```

Comment:
would it make more sense to use some of the values exported by the builder package like https://github.com/thanos-community/thanos-operator/blob/main/internal/pkg/manifests/query/builder.go#L19 ?